backends/gdx-backend-android/src/com/badlogic/gdx/backends/android/AndroidApplication.java

```diff
@@ -186,6 +191,14 @@ protected void createWakeLock (boolean use) {
 		}
 	}
 
+	@TargetApi(Build.VERSION_CODES.P)
+	private void tryRenderUnderCutout(boolean render) {
```

Comment:
I've tested the PR using libGDX tests and it seems to be working. 

Just a final change (sorry for being nitpicky). Can you change the name of the method `tryRenderUnderCutout` to something more descriptive such as setLayoutInDisplayCutoutMode(boolean)`.

Reply:
No problem. Sorry for the late response